minicluster/src/main/java/tachyon/underfs/UnderFileSystemCluster.java

```diff
@@ -51,6 +51,16 @@ public void run() {
   private static String sUfsClz;
 
   /**
+   * @return the existing underfs, throwing IllegalStateException if it hasn't been initialized yet
+   */
+  public static synchronized UnderFileSystemCluster get() {
```

Comment:
I've created https://tachyon.atlassian.net/browse/TACHYON-1233 for adding unit tests to `UnderFileSystemCluster`.